### PR TITLE
#16 view GitHub contributors(new)

### LINKED ISSUE #16: FEATURE - View GitHub Contributors
## Describe the feature you want the app to have that it currently doesn't
* Create a new section in the `SettingsView` called "Contributions" to view the list of contributors to the app, using the GitHub API (no login required)
* See how it was implemented [here](https://github.com/mikaelacaron/brain-marks/blob/025cbbb20df75c958df426bb63866cfce757b5e1/brain-marks/Settings/SettingsViewModel.swift#L13), in one of my archived repos

## Describe alternatives you've considered
* N/A

## Additional context
* Load the usernames and avatars

### PR DESCRIPTION
**Pull request by Yashraj49**
Contributors

# What it Does
* Implements 'Contributions' Section in SettingsView to keep track of contributors to the project.
* Closes #16 

# How I Tested
* Ran the application.
* Navigated to the SettingsView.
* Observed the new 'Contributions' section.
* Verified that the list of contributors is displayed.
* Tested the functionality of the 'Contributions' section by interacting with it.
* Checked that the GitHub API is fetching and displaying contributors correctly.

# Notes
* I haven't included my team ID, as I have not yet enrolled in Apple Dev Program

# Screenshots 
<div style="display: flex; flex-direction: row;">

<div>
  Simulator Screenshot - iPhone 15 Pro - 2023-10-04 at 19 19 48
  
  Simulator Screenshot - iPhone 15 Pro Max
</div>

### CODE REVIEW

**Comment by Yashraj49** [comment]
@mikaelacaron  I have fixed the required conflicts you may review it

File: Basic-Car-Maintenance.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -380,6 +392,7 @@
 			buildActionMask = 2147483647;
 			files = (
 				FF5D13AF2A86C2D800BC9BD6 /* Preview Assets.xcassets in Resources */,
+				E5C46DD72ACD2D4B0098D448 /* Basic-Car-Maintenance.xcconfig in Resources */,
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance.xcodeproj/project.pbxproj:395]: You can see here something is still happening with the `xcconfig` file moving around, can you make sure this is set up correctly and doesn't move? Thanks! If you have any issues DM me on social media

**Review by mikaelacaron — CHANGES_REQUESTED** [review]
It looks like the `xcconfig` is still moved in the project

File: Basic-Car-Maintenance.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Yashraj49) [reply Basic-Car-Maintenance.xcodeproj/project.pbxproj:395]: I didn't moved it . also I have no idea how to move it what is needed to be done here I mean is there any way to set it back ??  also this is first time I am encountering this issue

Comment (mikaelacaron) [reply Basic-Car-Maintenance.xcodeproj/project.pbxproj:395]: When you cloned the repo for the first time, did you run the terminal command to copy the file, or did you do File > New in Xcode to make the `xcconfig`?

Comment (Yashraj49) [reply Basic-Car-Maintenance.xcodeproj/project.pbxproj:395]: yes I did ,at the very first I did ran that command using terminal to its directory and then that file was created there

Comment (mikaelacaron) [reply Basic-Car-Maintenance.xcodeproj/project.pbxproj:395]: If you ran the command the then the xcconfig shouldn't have changed at all, in the pbxproj file
Can you try to just clone the `dev` branch again (just in a different place than you've been working in) and run the command again, and see if the pbxproj file has changed?

Comment (Yashraj49) [reply Basic-Car-Maintenance.xcodeproj/project.pbxproj:395]: I will sure do that but how's that going to help in now getting the file as before it was ??

Comment (mikaelacaron) [reply Basic-Car-Maintenance.xcodeproj/project.pbxproj:395]: 1st it confirms whether that is how it works, and 2nd, you can copy code from your PR over to that branch, if it works correctly (if you want), or the other solution is properly resolving the current merge conflicts

Comment (Yashraj49) [reply Basic-Car-Maintenance.xcodeproj/project.pbxproj:395]: ok that's fine and how can I properly resolve the current merge conflict ??

Comment (mikaelacaron) [reply Basic-Car-Maintenance.xcodeproj/project.pbxproj:395]: Look into git tutorials and how to solve merge conflicts!

Comment (Yashraj49) [reply Basic-Car-Maintenance.xcodeproj/project.pbxproj:395]: hey I have cloned dev branch again also I noticed that bundle id section is empty so should I let remain like that or should I paste your bundle id over there

**Comment by mikaelacaron** [comment]
Replaced by #70

**Pull request closed without merging**